app/Spago.hs

```diff
@@ -97,16 +98,25 @@ makeConfig force = do
 --   - create an example `test` folder
 initProject :: Bool -> IO ()
 initProject force = do
+  -- packages.dhall and spago.dhall overwrite can be forced
   PscPackage.makePackagesDhall force "init"
   makeConfig force
   T.mktree "src"
   T.mktree "test"
-  -- TODO fail if these files exist
-  T.writeTextFile "src/Main.purs" Templates.srcMain
-  T.writeTextFile "test/Main.purs" Templates.testMain
-  T.writeTextFile ".gitignore" Templates.gitignore
+  -- But the other files in the template are just skipped if already there.
+  -- Because you might want to just init a project with your own source files,
+  -- or just migrate a psc-package project
+  copyIfNotThere "src/Main.purs" Templates.srcMain
```

Comment:
IfNotExists

Reply:
Thank you, I knew there was a more fitting name 😄